ext/dfe_alpha_makefile_stdpopsim_patch

```diff
@@ -0,0 +1,18 @@
+GSLDIR = ${CONDA_PREFIX}
+
+CC        = gcc
+CFLAGS   += -lm -lgsl -lgslcblas -O4 -I$(GSLDIR)/include -L${GSLDIR}/lib
+
+all: est_dfe est_alpha_omega prop_muts_in_s_ranges
+       
+est_dfe : est_dfe.c routine_library.o bctmat01.c genlib.o nrlib.o nrutil.o tmatrix_routines.o
+	$(CC) -o $@ $^ $(CFLAGS)
+
+est_alpha_omega : est_alpha_omega.c routine_library.o bctmat01.o tmatrix_routines.o genlib.o seqlib.o nrlib.o nrutil.o 
+	$(CC) -o $@  $^ $(CFLAGS)
+       
+prop_muts_in_s_ranges : prop_muts_in_s_ranges.c routine_library.o genlib.o
+	$(CC) -o $@ $^ $(CFLAGS)
+
+clean:
+	@rm -f est_dfe est_alpha_omega prop_muts_in_s_ranges *.o 
```

Comment:
@andrewkern here is the Makefile for compiling DFE-alpha under the conda environment